config.json

```
@@ -1576,6 +1590,16 @@
       "uuid": "0455ecbd-dc84-4799-b49e-bc9bf0a9fc71",
       "slug": "iteration",
       "name": "Iteration"
+    },
+    {
+      "uuid": "a36e93cb-5ab0-4693-a6ab-ce698a3550ce",
+      "slug": "tuples",
+      "name": "Tuples"
```

```suggestion
      "slug": "tuple",
      "name": "Tuple"
```

Should this be singular or plural? `Symbol` is singular below and `Array` is singular above, but some of the concepts are plural.